bookmarks/images/tests/test_views.py

```diff
@@ -18,3 +16,42 @@ def test_detail_view_returns_404_for_invalid_image(self):
             reverse("images:detail", args=[1000, "image-illai"])
         )
         self.assertEquals(response.status_code, 404)
+
+
+class TestImageLikeView(ModelMixin, TestCase):
+    def test_image_like_view_succeeds_adding_like_to_the_image(self):
+        self.client.login(**self.credentials)
+        self.client.get(
+            reverse(
+                "images:detail",
+                args=[self.image.pk, self.image.slug],
+            )
+        )
+        self.image.users_like.add(self.user)
+        self.assertIsNotNone(self.image.users_like.first())
```

Comment:
Incorrect. Please call client.post or request factory. Check if the user is added in users_like.